data/src/main/java/org/oppia/android/data/backends/gae/NetworkModule.kt

```diff
@@ -31,7 +32,8 @@ class NetworkModule {
   @Singleton
   fun provideRetrofitInstance(): Retrofit {
     val client = OkHttpClient.Builder()
-    client.addInterceptor(NetworkInterceptor())
+    client.addInterceptor(JsonPrefixNetworkInterceptor())
+      .addInterceptor(RemoteAuthNetworkInterceptor())
```

Comment:
These should be parameter injected, not constructed, otherwise their injections won't be ready.

You can do that by adding these as parameters to the provider method.

Reply:
Done